extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Layout} from '../../../src/layout';
+import {assert} from '../../../src/asserts';
+
+class AmpAccordion extends AMP.BaseElement {
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return layout == Layout.CONTAINER;
+  }
+
+  /** @override */
+  buildCallback() {
+    /** @const @private {!NodeList} */
+    this.sections_ = this.element.children;
```

Comment:
Change to `this.getRealChildren()`. As an additional benefit this will return an array which would be easy to iterate without `forEach.call` and such.